server/store/store.go

```diff
@@ -685,6 +686,17 @@ func (s *SQLStore) GetChannelSubscription(subscriptionID string) (*storemodels.C
 	return &subscription, nil
 }
 
+func (s *SQLStore) GetChannelSubscriptionByTeamsChannelID(teamsChannelID string) (*storemodels.ChannelSubscription, error) {
+	row := s.getQueryBuilder().Select("subscriptionID").From("msteamssync_subscriptions").Where(sq.Eq{"msTeamsChannelID": teamsChannelID, "type": subscriptionTypeChannel}).QueryRow()
+	var subscription storemodels.ChannelSubscription
+	var expiresOn int64
+	if scanErr := row.Scan(&subscription.SubscriptionID); scanErr != nil {
+		return nil, scanErr
+	}
+	subscription.ExpiresOn = time.UnixMicro(expiresOn)
```

Comment:
what's the use of this?

Reply:
It was copied from the above function `GetChannelSubscription`. It is used to convert `int64` to `time.Time`.

Reply:
i think we can remove it as it is not used anywhere